src/booleans.js

```diff
@@ -1,63 +1,123 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  if (a === true && b === true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function either(a, b) {
-  // your code here
+  if (a === true || b === true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function none(a, b) {
-  // your code here
+  if (a !== true && b !== true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function one(a, b) {
-  // your code here
+  if (a === true && b === false || a === false && b === true) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
Can be simplified as above:
```suggestion
  return (a && !b) || (!a && b);
```